src/utils/libs/requestApi.ts

```diff
@@ -1,19 +1,25 @@
 import axios, { AxiosRequestConfig } from "axios";
+import errHandler from "./error";
 
-const RequestApiV2 = (p: AxiosRequestConfig) => {
+export interface requestApiV2DTO extends AxiosRequestConfig {
+  canHeader?: boolean;
+}
+const RequestApiV2 = (p: requestApiV2DTO) => {
   try {
     const res = axios({
       method: p.method,
       baseURL: process.env.NEXT_PUBLIC_APP_BASE_URLV2,
       url: p.url,
       data: p.data,
-      headers: p.headers
-        ? { Authorization: localStorage.getItem("token") }
-        : null,
+      headers: p.canHeader
```

Comment:
```suggestion
      headers: Object.assign({}, p.canHeader ? { Authorization: localStorage.getItem("token") ?? "" } : {}, p.headers);
```
헤더를 주입하여 원하는 헤더도 사용가능하도록 하는 것이 추후 유지보수가 편하지않을까 싶습니다.